pipelines/nextflow/modules/gff3/process_gff3.nf

```diff
@@ -20,7 +20,7 @@ process PROCESS_GFF3 {
     label 'cached'
 
     input:
-        tuple val(meta), path(gff3), path(genome)
+        tuple val(meta), path(genome), path(gff3)
```

Comment:
I hope, this is supported both in `genome_prepare` and `additional_seq_prepare subworkflows`. I see the patch. Just confirming.

Reply:
Yes, I have tried to be thorough and any changes in the `input` or `output` are applied to every pipeline affected.